const e=[{Key:"ST8N3YKE","Item Type":"preprint","Publication Year":2022,Author:"Cragnolini, Tristan; Beton, Joseph; Topf, Maya",Title:"Cryo-EM structure and B-factor refinement with ensemble representation","Publication Title":null,ISBN:null,ISSN:null,DOI:"10.1101/2022.06.08.495259",Url:"https://www.biorxiv.org/content/10.1101/2022.06.08.495259v1","Abstract Note":"Cryo-EM experiments produce images of macromolecular assemblies that are combined to produce three-dimensional density maps. It is common to fit atomic models of the contained molecules to interpret those maps, followed by a density-guided refinement. Here, we propose TEMPy-REFF, a novel method for atomic structure refinement in cryo-EM density maps. By representing the atomic positions as components of a mixture model, their variances as B-factors, and a model ensemble description, we significantly improve the fit to the map compared to what is currently achievable with state-of-the-art methods. We validate our method on a large benchmark of 366 cryo-EM maps from EMDB at 1.8-7.1Å resolution and their corresponding PDB assembly models. We also show that our approach can provide newly-modelled regions in EMDB deposited maps by combining it with AlphaFold-Multimer. Finally, our method provides a natural interpretation of maps into components, allowing us to accurately create composite maps.",Date:"2022-06-09","Date Added":"2022-06-12 14:27:25","Date Modified":"2023-06-16 12:59:25","Access Date":"2022-08-19 13:34:50",Pages:null,"Num Pages":null,Issue:null,Volume:null,"Number Of Volumes":null,"Journal Abbreviation":null,"Short Title":null,Series:null,"Series Number":null,"Series Text":null,"Series Title":null,Publisher:"bioRxiv",Place:null,Language:"en",Rights:"© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/",Type:null,Archive:null,"Archive Location":null,"Library Catalog":"bioRxiv","Call Number":null,Extra:"Pages: 2022.06.08.495259 Section: New Results",Notes:null,"File Attachments":"/Users/Tristan/Zotero/storage/QWK3Z4IS/Cragnolini et al. - 2022 - Cryo-EM structure and B-factor refinement with ens.pdf","Link Attachments":null,"Manual Tags":null,"Automatic Tags":null,Editor:null,"Series Editor":null,Translator:null,Contributor:null,"Attorney Agent":null,"Book Author":null,"Cast Member":null,Commenter:null,Composer:null,Cosponsor:null,Counsel:null,Interviewer:null,Producer:null,Recipient:null,"Reviewed Author":null,Scriptwriter:null,"Words By":null,Guest:null,Number:null,Edition:null,"Running Time":null,Scale:null,Medium:null,"Artwork Size":null,"Filing Date":null,"Application Number":null,Assignee:null,"Issuing Authority":null,Country:null,"Meeting Name":null,"Conference Name":null,Court:null,References:null,Reporter:null,"Legal Status":null,"Priority Numbers":null,"Programming Language":null,Version:null,System:null,Code:null,"Code Number":null,Section:null,Session:null,Committee:null,History:null,"Legislative Body":null,authors:"Cragnolini, Tristan; Beton, Joseph; Topf, Maya"},{Key:"SDI7ZE5X","Item Type":"journalArticle","Publication Year":2022,Author:"Britt, Hannah M.; Cragnolini, Tristan; Khatun, Suniya; Hatimy, Abubakar; James, Juliette; Page, Nathanael; Williams, Jonathan P.; Hughes, Christopher; Denny, Richard; Thalassinos, Konstantinos; Vissers, Johannes P. C.",Title:"Evaluation of Acquisition Modes for Semi‐Quantitative Analysis by Targeted and Untargeted Mass Spectrometry","Publication Title":"Rapid Communications in Mass Spectrometry",ISBN:null,ISSN:"0951-4198",DOI:"10.1002/rcm.9308",Url:null,"Abstract Note":null,Date:"2022","Date Added":"2022-05-23 09:46:01","Date Modified":"2022-05-23 11:42:50","Access Date":null,Pages:"1-13","Num Pages":null,Issue:"March",Volume:null,"Number Of Volumes":null,"Journal Abbreviation":null,"Short Title":null,Series:null,"Series Number":null,"Series Text":null,"Series Title":null,Publisher:null,Place:null,Language:null,Rights:"All rights reserved",Type:null,Archive:null,"Archive Location":null,"Library Catalog":null,"Call Number":null,Extra:null,Notes:null,"File Attachments":"/Users/Tristan/Zotero/storage/EQRK3K79/Britt et al. - 2022.pdf","Link Attachments":null,"Manual Tags":null,"Automatic Tags":null,Editor:null,"Series Editor":null,Translator:null,Contributor:null,"Attorney Agent":null,"Book Author":null,"Cast Member":null,Commenter:null,Composer:null,Cosponsor:null,Counsel:null,Interviewer:null,Producer:null,Recipient:null,"Reviewed Author":null,Scriptwriter:null,"Words By":null,Guest:null,Number:null,Edition:null,"Running Time":null,Scale:null,Medium:null,"Artwork Size":null,"Filing Date":null,"Application Number":null,Assignee:null,"Issuing Authority":null,Country:null,"Meeting Name":null,"Conference Name":null,Court:null,References:null,Reporter:null,"Legal Status":null,"Priority Numbers":null,"Programming Language":null,Version:null,System:null,Code:null,"Code Number":null,Section:null,Session:null,Committee:null,History:null,"Legislative Body":null,authors:"Britt, Hannah M.; Cragnolini, Tristan; Khatun, Suniya; Hatimy, Abubakar; James, Juliette; Page, Nathanael; Williams, Jonathan P.; Hughes, Christopher; Denny, Richard; Thalassinos, Konstantinos; Vissers, Johannes P. C."},{Key:"P2MPXFNE","Item Type":"journalArticle","Publication Year":2022,Author:"Britt, Hannah M.; Cragnolini, Tristan; Thalassinos, Konstantinos",Title:"Integration of Mass Spectrometry Data for Structural Biology","Publication Title":"Chemical Reviews",ISBN:null,ISSN:"0009-2665",DOI:"10.1021/acs.chemrev.1c00356",Url:"https://pubs.acs.org/doi/10.1021/acs.chemrev.1c00356","Abstract Note":"Mass spectrometry (MS) is increasingly being used to probe the structure and dynamics of proteins and the complexes they form with other macromolecules. There are now several specialized MS methods...",Date:"2022-04-27","Date Added":"2021-09-29 16:03:49","Date Modified":"2022-06-12 14:27:26","Access Date":null,Pages:"7952-7986","Num Pages":null,Issue:"8",Volume:122,"Number Of Volumes":null,"Journal Abbreviation":null,"Short Title":null,Series:null,"Series Number":null,"Series Text":null,"Series Title":null,Publisher:null,Place:null,Language:null,Rights:"All rights reserved",Type:null,Archive:null,"Archive Location":null,"Library Catalog":null,"Call Number":null,Extra:"Citation Key: Britt2022",Notes:null,"File Attachments":"/Users/Tristan/Zotero/storage/9GM4I2C4/Britt, Cragnolini, Thalassinos - 2021.pdf","Link Attachments":null,"Manual Tags":null,"Automatic Tags":null,Editor:null,"Series Editor":null,Translator:null,Contributor:null,"Attorney Agent":null,"Book Author":null,"Cast Member":null,Commenter:null,Composer:null,Cosponsor:null,Counsel:null,Interviewer:null,Producer:null,Recipient:null,"Reviewed Author":null,Scriptwriter:null,"Words By":null,Guest:null,Number:null,Edition:null,"Running Time":null,Scale:null,Medium:null,"Artwork Size":null,"Filing Date":null,"Application Number":null,Assignee:null,"Issuing Authority":null,Country:null,"Meeting Name":null,"Conference Name":null,Court:null,References:null,Reporter:null,"Legal Status":null,"Priority Numbers":null,"Programming Language":null,Version:null,System:null,Code:null,"Code Number":null,Section:null,Session:null,Committee:null,History:null,"Legislative Body":null,authors:"Britt, Hannah M.; Cragnolini, Tristan; Thalassinos, Konstantinos"},{Key:"YXNR8GBB","Item Type":"journalArticle","Publication Year":2021,Author:"Eldrid, Charles; Ben-Younis, Aisha; Ujma, Jakub; Britt, Hannah; Cragnolini, Tristan; Kalfas, Symeon; Cooper-Shepherd, Dale; Tomczyk, Nick; Giles, Kevin; Morris, Mike; Akter, Rehana; Raleigh, Daniel; Thalassinos, Konstantinos",Title:"Cyclic Ion Mobility–Collision Activation Experiments Elucidate Protein Behavior in the Gas Phase","Publication Title":"Journal of the American Society for Mass Spectrometry",ISBN:null,ISSN:"1044-0305",DOI:"10.1021/jasms.1c00018",Url:"https://pubs.acs.org/doi/10.1021/jasms.1c00018","Abstract Note":"Ion mobility coupled to mass spectrometry (IM-MS) is widely used to study protein dynamics and structure in the gas phase. Increasing the energy with which the protein ions are introduced to the IM cell can induce them to unfold, providing information on the comparative energetics of unfolding between different proteoforms. Recently, a high-resolution cyclic IM-mass spectrometer (cIM-MS) was introduced, allowing multiple, consecutive tandem IM experiments (IMn) to be carried out. We describe a tandem IM technique for defining detailed protein unfolding pathways and the dynamics of disordered proteins. The method involves multiple rounds of IM separation and collision activation (CA): IM-CA-IM and CA-IM-CA-IM. Here, we explore its application to studies of a model protein, cytochrome C, and dimeric human islet amyloid polypeptide (hIAPP), a cytotoxic and amyloidogenic peptide involved in type II diabetes. In agreement with prior work using single stage IM-MS, several unfolding events are observed for cytochrome C. IMn-MS experiments also show evidence of interconversion between compact and extended structures. IMn-MS data for hIAPP shows interconversion prior to dissociation, suggesting that the certain conformations have low energy barriers between them and transition between compact and extended forms.",Date:"2021-06-02","Date Added":"2022-03-08 08:55:33","Date Modified":"2022-06-15 11:44:16","Access Date":null,Pages:"1545-1552","Num Pages":null,Issue:"6",Volume:32,"Number Of Volumes":null,"Journal Abbreviation":null,"Short Title":null,Series:null,"Series Number":null,"Series Text":null,"Series Title":null,Publisher:null,Place:null,Language:null,Rights:"All rights reserved",Type:null,Archive:null,"Archive Location":null,"Library Catalog":null,"Call Number":null,Extra:"PMID: 34006100 Citation Key: Eldrid2021",Notes:null,"File Attachments":"/Users/Tristan/Zotero/storage/EHAYS2WA/Eldrid et al. - 2021.pdf","Link Attachments":null,"Manual Tags":null,"Automatic Tags":"ion-mobility mass spectrometry; protein unfolding",Editor:null,"Series Editor":null,Translator:null,Contributor:null,"Attorney Agent":null,"Book Author":null,"Cast Member":null,Commenter:null,Composer:null,Cosponsor:null,Counsel:null,Interviewer:null,Producer:null,Recipient:null,"Reviewed Author":null,Scriptwriter:null,"Words By":null,Guest:null,Number:null,Edition:null,"Running Time":null,Scale:null,Medium:null,"Artwork Size":null,"Filing Date":null,"Application Number":null,Assignee:null,"Issuing Authority":null,Country:null,"Meeting Name":null,"Conference Name":null,Court:null,References:null,Reporter:null,"Legal Status":null,"Priority Numbers":null,"Programming Language":null,Version:null,System:null,Code:null,"Code Number":null,Section:null,Session:null,Committee:null,History:null,"Legislative Body":null,authors:"Eldrid, Charles; Ben-Younis, Aisha; Ujma, Jakub; Britt, Hannah; Cragnolini, Tristan; Kalfas, Symeon; Cooper-Shepherd, Dale; Tomczyk, Nick; Giles, Kevin; Morris, Mike; Akter, Rehana; Raleigh, Daniel; Thalassinos, Konstantinos"},{Key:"IXZFN7DZ","Item Type":"journalArticle","Publication Year":2021,Author:"Cragnolini, Tristan; Kryshtafovych, Andriy; Topf, Maya",Title:"Cryo-EM targets in CASP14","Publication Title":"Proteins",ISBN:null,ISSN:"1097-0134",DOI:"10.1002/prot.26216",Url:"https://onlinelibrary.wiley.com/doi/10.1002/prot.26216","Abstract Note":'Structures of seven CASP14 targets were determined using cryo-electron microscopy (cryo-EM) technique with resolution between 2.1 and 3.8 Å. We provide an evaluation of the submitted models versus the experimental data (cryo-EM density maps) and experimental reference structures built into the maps. The accuracy of models is measured in terms of coordinate-to-density and coordinate-to-coordinate fit. A-posteriori refinement of the most accurate models in their corresponding cryo-EM density resulted in structures that are close to the reference structure, including some regions with better fit to the density. Regions that were found to be less "refineable" correlate well with regions of high diversity between the CASP models and low goodness-of-fit to density in the reference structure.',Date:"2021-12-16","Date Added":"2022-02-15 15:46:42","Date Modified":"2022-06-04 11:50:05","Access Date":null,Pages:"1949-1958","Num Pages":null,Issue:"12",Volume:89,"Number Of Volumes":null,"Journal Abbreviation":null,"Short Title":null,Series:null,"Series Number":null,"Series Text":null,"Series Title":null,Publisher:null,Place:null,Language:null,Rights:"All rights reserved",Type:null,Archive:null,"Archive Location":null,"Library Catalog":null,"Call Number":null,Extra:"PMID: 34398978 Citation Key: Cragnolini2021",Notes:null,"File Attachments":"; ; /Users/Tristan/Zotero/storage/T4NRJ25D/Cragnolini, Kryshtafovych, Topf - 2021.pdf","Link Attachments":"http://www.ncbi.nlm.nih.gov/pubmed/34398978; http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC8630773","Manual Tags":null,"Automatic Tags":"CASP; electron microscopy; protein structure prediction; cryo-EM; model evaluation",Editor:null,"Series Editor":null,Translator:null,Contributor:null,"Attorney Agent":null,"Book Author":null,"Cast Member":null,Commenter:null,Composer:null,Cosponsor:null,Counsel:null,Interviewer:null,Producer:null,Recipient:null,"Reviewed Author":null,Scriptwriter:null,"Words By":null,Guest:null,Number:null,Edition:null,"Running Time":null,Scale:null,Medium:null,"Artwork Size":null,"Filing Date":null,"Application Number":null,Assignee:null,"Issuing Authority":null,Country:null,"Meeting Name":null,"Conference Name":null,Court:null,References:null,Reporter:null,"Legal Status":null,"Priority Numbers":null,"Programming Language":null,Version:null,System:null,Code:null,"Code Number":null,Section:null,Session:null,Committee:null,History:null,"Legislative Body":null,authors:"Cragnolini, Tristan; Kryshtafovych, Andriy; Topf, Maya"},{Key:"6IKF4RTZ","Item Type":"journalArticle","Publication Year":2021,Author:"Cragnolini, Tristan; Sahota, Harpal; Joseph, Agnel Praveen; Sweeney, Aaron; Malhotra, Sony; Vasishtan, Daven; Topf, Maya",Title:"TEMPy 2: a Python library with improved 3D electron microscopy density-fitting and validation workflows","Publication Title":"Acta Crystallographica Section D Structural Biology",ISBN:null,ISSN:"2059-7983",DOI:"10.1107/S2059798320014928",Url:"https://scripts.iucr.org/cgi-bin/paper?S2059798320014928","Abstract Note":"Structural determination of molecular complexes by cryo-EM requires large, often complex processing of the image data that are initially obtained. Here, TEMPy 2, an update of the TEMPy package to process, optimize and assess cryo-EM maps and the structures fitted to them, is described. New optimization routines, comprehensive automated checks and workflows to perform these tasks are described.",Date:"2021-01-01","Date Added":"2022-03-08 08:55:33","Date Modified":"2022-03-14 11:57:17","Access Date":null,Pages:"41-47","Num Pages":null,Issue:"1",Volume:77,"Number Of Volumes":null,"Journal Abbreviation":null,"Short Title":null,Series:null,"Series Number":null,"Series Text":null,"Series Title":null,Publisher:null,Place:null,Language:null,Rights:"All rights reserved",Type:null,Archive:null,"Archive Location":null,"Library Catalog":null,"Call Number":null,Extra:"PMID: 33404524",Notes:null,"File Attachments":"/Users/Tristan/Zotero/storage/3QSGKYXP/Cragnolini et al. - 2021.pdf","Link Attachments":null,"Manual Tags":null,"Automatic Tags":"validation; macromolecular complexes; fitting scores; model assessment; model fitting; TEMPy2; three-dimensional electron microscopy",Editor:null,"Series Editor":null,Translator:null,Contributor:null,"Attorney Agent":null,"Book Author":null,"Cast Member":null,Commenter:null,Composer:null,Cosponsor:null,Counsel:null,Interviewer:null,Producer:null,Recipient:null,"Reviewed Author":null,Scriptwriter:null,"Words By":null,Guest:null,Number:null,Edition:null,"Running Time":null,Scale:null,Medium:null,"Artwork Size":null,"Filing Date":null,"Application Number":null,Assignee:null,"Issuing Authority":null,Country:null,"Meeting Name":null,"Conference Name":null,Court:null,References:null,Reporter:null,"Legal Status":null,"Priority Numbers":null,"Programming Language":null,Version:null,System:null,Code:null,"Code Number":null,Section:null,Session:null,Committee:null,History:null,"Legislative Body":null,authors:"Cragnolini, Tristan; Sahota, Harpal; Joseph, Agnel Praveen; Sweeney, Aaron; Malhotra, Sony; Vasishtan, Daven; Topf, Maya"},{Key:"2HTW2VL2","Item Type":"conferencePaper","Publication Year":2020,Author:"Cragnolini, Tristan; Eldrid, Charles; Britt, Hannah M.; Menneteau, Thomas; Ben-Younis, Aisha; Thalassinos, Konstantinos",Title:"Automated, web-based analysis and visualisation of tandem ion mobility mass spectrometry data","Publication Title":"Proceedings of the 68th ASMS Conference on Mass Spectrometry and Allied Topics",ISBN:null,ISSN:null,DOI:null,Url:null,"Abstract Note":null,Date:"2020","Date Added":"2020-11-11 17:59:58","Date Modified":"2020-11-11 18:00:00","Access Date":null,Pages:null,"Num Pages":null,Issue:null,Volume:null,"Number Of Volumes":null,"Journal Abbreviation":null,"Short Title":null,Series:null,"Series Number":null,"Series Text":null,"Series Title":null,Publisher:null,Place:"Online Meeting",Language:null,Rights:"All rights reserved",Type:null,Archive:null,"Archive Location":null,"Library Catalog":null,"Call Number":null,Extra:null,Notes:null,"File Attachments":null,"Link Attachments":null,"Manual Tags":null,"Automatic Tags":null,Editor:null,"Series Editor":null,Translator:null,Contributor:null,"Attorney Agent":null,"Book Author":null,"Cast Member":null,Commenter:null,Composer:null,Cosponsor:null,Counsel:null,Interviewer:null,Producer:null,Recipient:null,"Reviewed Author":null,Scriptwriter:null,"Words By":null,Guest:null,Number:null,Edition:null,"Running Time":null,Scale:null,Medium:null,"Artwork Size":null,"Filing Date":null,"Application Number":null,Assignee:null,"Issuing Authority":null,Country:null,"Meeting Name":null,"Conference Name":null,Court:null,References:null,Reporter:null,"Legal Status":null,"Priority Numbers":null,"Programming Language":null,Version:null,System:null,Code:null,"Code Number":null,Section:null,Session:null,Committee:null,History:null,"Legislative Body":null,authors:"Cragnolini, Tristan; Eldrid, Charles; Britt, Hannah M.; Menneteau, Thomas; Ben-Younis, Aisha; Thalassinos, Konstantinos"},{Key:"4V2C76WS","Item Type":"journalArticle","Publication Year":2019,Author:"Kimchi, Ofer; Cragnolini, Tristan; Brenner, Michael P.; Colwell, Lucy J.",Title:"A Polymer Physics Framework for the Entropy of Arbitrary Pseudoknots","Publication Title":"Biophysical Journal",ISBN:null,ISSN:"15420086",DOI:"10.1016/j.bpj.2019.06.037",Url:"https://doi.org/10.1016/j.bpj.2019.06.037","Abstract Note":"The accurate prediction of RNA secondary structure from primary sequence has had enormous impact on research from the past 40 years. Although many algorithms are available to make these predictions, the inclusion of non-nested loops, termed pseudoknots, still poses challenges arising from two main factors: 1) no physical model exists to estimate the loop entropies of complex intramolecular pseudoknots, and 2) their NP-complete enumeration has impeded their study. Here, we address both challenges. First, we develop a polymer physics model that can address arbitrarily complex pseudoknots using only two parameters corresponding to concrete physical quantities—over an order of magnitude fewer than the sparsest state-of-the-art phenomenological methods. Second, by coupling this model to exhaustive enumeration of the set of possible structures, we compute the entire free energy landscape of secondary structures resulting from a primary RNA sequence. We demonstrate that for RNA structures of ∼80 nucleotides, with minimal heuristics, the complete enumeration of possible secondary structures can be accomplished quickly despite the NP-complete nature of the problem. We further show that despite our loop entropy model's parametric sparsity, it performs better than or on par with previously published methods in predicting both pseudoknotted and non-pseudoknotted structures on a benchmark data set of RNA structures of ≤80 nucleotides. We suggest ways in which the accuracy of the model can be further improved.",Date:"2019","Date Added":"2022-05-23 09:46:01","Date Modified":"2022-05-23 09:46:42","Access Date":null,Pages:"520-532","Num Pages":null,Issue:"3",Volume:117,"Number Of Volumes":null,"Journal Abbreviation":null,"Short Title":null,Series:null,"Series Number":null,"Series Text":null,"Series Title":null,Publisher:null,Place:null,Language:null,Rights:"All rights reserved",Type:null,Archive:null,"Archive Location":null,"Library Catalog":null,"Call Number":null,Extra:"PMID: 31353036 Publisher: Biophysical Society",Notes:null,"File Attachments":"/Users/Tristan/Zotero/storage/7HMUMSXP/Kimchi et al. - 2019.pdf","Link Attachments":null,"Manual Tags":null,"Automatic Tags":null,Editor:null,"Series Editor":null,Translator:null,Contributor:null,"Attorney Agent":null,"Book Author":null,"Cast Member":null,Commenter:null,Composer:null,Cosponsor:null,Counsel:null,Interviewer:null,Producer:null,Recipient:null,"Reviewed Author":null,Scriptwriter:null,"Words By":null,Guest:null,Number:null,Edition:null,"Running Time":null,Scale:null,Medium:null,"Artwork Size":null,"Filing Date":null,"Application Number":null,Assignee:null,"Issuing Authority":null,Country:null,"Meeting Name":null,"Conference Name":null,Court:null,References:null,Reporter:null,"Legal Status":null,"Priority Numbers":null,"Programming Language":null,Version:null,System:null,Code:null,"Code Number":null,Section:null,Session:null,Committee:null,History:null,"Legislative Body":null,authors:"Kimchi, Ofer; Cragnolini, Tristan; Brenner, Michael P.; Colwell, Lucy J."},{Key:"TDCZXV62","Item Type":"journalArticle","Publication Year":2019,Author:"Kryshtafovych, Andriy; Malhotra, Sony; Monastyrskyy, Bohdan; Cragnolini, Tristan; Joseph, Agnel-praveen; Chiu, Wah; Topf, Maya",Title:"Cryo‐electron microscopy targets in CASP13: Overview and evaluation of results","Publication Title":"Proteins: Structure, Function, and Bioinformatics",ISBN:null,ISSN:"0887-3585",DOI:"10.1002/prot.25817",Url:"https://onlinelibrary.wiley.com/doi/abs/10.1002/prot.25817","Abstract Note":null,Date:"2019-12-23","Date Added":"2019-10-31 14:47:48","Date Modified":"2022-03-14 11:57:17","Access Date":null,Pages:"1128-1140","Num Pages":null,Issue:"12",Volume:87,"Number Of Volumes":null,"Journal Abbreviation":null,"Short Title":null,Series:null,"Series Number":null,"Series Text":null,"Series Title":null,Publisher:null,Place:null,Language:null,Rights:"All rights reserved",Type:null,Archive:null,"Archive Location":null,"Library Catalog":null,"Call Number":null,Extra:null,Notes:null,"File Attachments":"; /Users/Tristan/Zotero/storage/H39URCRP/Kryshtafovych_et_al-2019-Proteins__Structure,_Function,_and_Bioinformatics.pdf","Link Attachments":"https://onlinelibrary.wiley.com/doi/10.1002/prot.25817","Manual Tags":null,"Automatic Tags":"1; electron microscopy; cryo-em; protein structure prediction; casp; cryogenic electron microscopy; d uc t i; i n t ro; is becoming increasingly; model evaluation; o n",Editor:null,"Series Editor":null,Translator:null,Contributor:null,"Attorney Agent":null,"Book Author":null,"Cast Member":null,Commenter:null,Composer:null,Cosponsor:null,Counsel:null,Interviewer:null,Producer:null,Recipient:null,"Reviewed Author":null,Scriptwriter:null,"Words By":null,Guest:null,Number:null,Edition:null,"Running Time":null,Scale:null,Medium:null,"Artwork Size":null,"Filing Date":null,"Application Number":null,Assignee:null,"Issuing Authority":null,Country:null,"Meeting Name":null,"Conference Name":null,Court:null,References:null,Reporter:null,"Legal Status":null,"Priority Numbers":null,"Programming Language":null,Version:null,System:null,Code:null,"Code Number":null,Section:null,Session:null,Committee:null,History:null,"Legislative Body":null,authors:"Kryshtafovych, Andriy; Malhotra, Sony; Monastyrskyy, Bohdan; Cragnolini, Tristan; Joseph, Agnel-praveen; Chiu, Wah; Topf, Maya"},{Key:"CNPWJB7H","Item Type":"journalArticle","Publication Year":2017,Author:"Cragnolini, Tristan; Chakraborty, Debayan; Šponer, Jiří; Derreumaux, Philippe; Pasquali, Samuela; Wales, David J.",Title:"Multifunctional energy landscape for a DNA G-quadruplex: An evolved molecular switch","Publication Title":"Journal of Chemical Physics",ISBN:null,ISSN:"00219606",DOI:"10.1063/1.4997377",Url:"http://aip.scitation.org/doi/10.1063/1.4997377","Abstract Note":"© 2017 Author(s). We explore the energy landscape for a four-fold telomere repeat, obtaining interconversion pathways between six experimentally characterised G-quadruplex topologies. The results reveal a multi-funnel system, with a variety of intermediate configurations and misfolded states. This organisation is identified with the intrinsically multi-functional nature of the system, suggesting a new paradigm for the classification of such biomolecules and clarifying issues regarding apparently conflicting experimental results.",Date:"2017-10-21","Date Added":"2017-10-02 12:47:20","Date Modified":"2018-05-29 10:49:09","Access Date":"2017-08-22",Pages:"152715","Num Pages":null,Issue:"15",Volume:147,"Number Of Volumes":null,"Journal Abbreviation":null,"Short Title":null,Series:null,"Series Number":null,"Series Text":null,"Series Title":null,Publisher:null,Place:null,Language:null,Rights:"All rights reserved",Type:null,Archive:null,"Archive Location":null,"Library Catalog":null,"Call Number":null,Extra:"Citation Key: Cragnolini2017",Notes:null,"File Attachments":"; ; /Users/Tristan/Zotero/storage/RNT8DN95/Cragnolini et al._2017.pdf","Link Attachments":"http://dx.doi.org/10.1063/1.4997377; http://aip.scitation.org/toc/jcp/147/15","Manual Tags":null,"Automatic Tags":null,Editor:null,"Series Editor":null,Translator:null,Contributor:null,"Attorney Agent":null,"Book Author":null,"Cast Member":null,Commenter:null,Composer:null,Cosponsor:null,Counsel:null,Interviewer:null,Producer:null,Recipient:null,"Reviewed Author":null,Scriptwriter:null,"Words By":null,Guest:null,Number:null,Edition:null,"Running Time":null,Scale:null,Medium:null,"Artwork Size":null,"Filing Date":null,"Application Number":null,Assignee:null,"Issuing Authority":null,Country:null,"Meeting Name":null,"Conference Name":null,Court:null,References:null,Reporter:null,"Legal Status":null,"Priority Numbers":null,"Programming Language":null,Version:null,System:null,Code:null,"Code Number":null,Section:null,Session:null,Committee:null,History:null,"Legislative Body":null,authors:"Cragnolini, Tristan; Chakraborty, Debayan; Šponer, Jiří; Derreumaux, Philippe; Pasquali, Samuela; Wales, David J."},{Key:"KTE9RCEH","Item Type":"journalArticle","Publication Year":2016,Author:"Stadlbauer, Petr; Mazzanti, Liuba; Cragnolini, Tristan; Wales, David J.; Derreumaux, Philippe; Pasquali, Samuela; Šponer, Jiří",Title:"Coarse-Grained Simulations Complemented by Atomistic Molecular Dynamics Provide New Insights into Folding and Unfolding of Human Telomeric G-Quadruplexes","Publication Title":"Journal of Chemical Theory and Computation",ISBN:null,ISSN:"15499626",DOI:"10.1021/acs.jctc.6b00667",Url:null,"Abstract Note":"G-quadruplexes are the most important noncanonical DNA architectures. Many quadruplex-forming sequences, including the human telomeric sequence d(GGGTTA)n, have been investigated due to their implications in cancer and other diseases, and because of their potential in DNA-based nanotechnology. Despite the availability of atomistic structural studies of folded G-quadruplexes, their folding pathways remain mysterious, and mutually contradictory models of folding coexist in the literature. Recent experiments convincingly demonstrated that G-quadruplex folding often takes days to reach thermodynamic equilibrium. Based on atomistic simulations of diverse classes of intermediates in G-quadruplex folding, we have suggested that the folding is an extremely multipathway process combining a kinetic partitioning mechanism with conformational diffusion. However, complete G-quadruplex folding is far beyond the time scale of atomistic simulations. Here we use high-resolution coarse-grained simulations to investigate po...",Date:"2016","Date Added":"2018-03-02 12:16:36","Date Modified":"2018-08-23 12:10:32","Access Date":"2017-08-29",Pages:"6077-6097","Num Pages":null,Issue:"12",Volume:12,"Number Of Volumes":null,"Journal Abbreviation":null,"Short Title":null,Series:null,"Series Number":null,"Series Text":null,"Series Title":null,Publisher:null,Place:null,Language:null,Rights:"All rights reserved",Type:null,Archive:null,"Archive Location":null,"Library Catalog":null,"Call Number":null,Extra:"PMID: 27767303 Citation Key: Stadlbauer2016",Notes:null,"File Attachments":"/Users/Tristan/Zotero/storage/4CD7Q4H4/m-api-6eb0488c-2808-fa8d-534b-62eba464e04e.pdf","Link Attachments":null,"Manual Tags":null,"Automatic Tags":null,Editor:null,"Series Editor":null,Translator:null,Contributor:null,"Attorney Agent":null,"Book Author":null,"Cast Member":null,Commenter:null,Composer:null,Cosponsor:null,Counsel:null,Interviewer:null,Producer:null,Recipient:null,"Reviewed Author":null,Scriptwriter:null,"Words By":null,Guest:null,Number:null,Edition:null,"Running Time":null,Scale:null,Medium:null,"Artwork Size":null,"Filing Date":null,"Application Number":null,Assignee:null,"Issuing Authority":null,Country:null,"Meeting Name":null,"Conference Name":null,Court:null,References:null,Reporter:null,"Legal Status":null,"Priority Numbers":null,"Programming Language":null,Version:null,System:null,Code:null,"Code Number":null,Section:null,Session:null,Committee:null,History:null,"Legislative Body":null,authors:"Stadlbauer, Petr; Mazzanti, Liuba; Cragnolini, Tristan; Wales, David J.; Derreumaux, Philippe; Pasquali, Samuela; Šponer, Jiří"},{Key:"WIGQNR5A","Item Type":"journalArticle","Publication Year":2016,Author:"Martinez-Zapien, Denise; Legrand, Pierre; McEwen, Alastair G. A.G.; Proux, Florence; Cragnolini, Tristan; Pasquali, Samuela; Dock-Bregeon, Anne Catherine A.-C.",Title:"The crystal structure of the 5' functional domain of the transcription riboregulator 7SK","Publication Title":"Nucleic Acids Research",ISBN:null,ISSN:"13624962",DOI:"10.1093/nar/gkw1351",Url:null,"Abstract Note":"In vertebrates, the 7SK RNA forms the scaffold of a complex, which regulates transcription pausing of RNA-polymerase II. By binding to the HEXIM protein, the complex comprising proteins LARP7 and MePCE captures the positive transcription elongation factor P-TEFb and prevents phosphorylation of pausing factors. The HEXIM-binding site embedded in the 5'-hairpin of 7SK (HP1) encompasses a short signature sequence, a GAUC repeat framed by single-stranded uridines. The present crystal structure of HP1 shows a remarkably straight helical stack involving several unexpected triples formed at a central region. Surprisingly, two uridines of the signature sequence make triple interactions in the major groove of the (GAUC)2 The third uridine is turned outwards or inward, wedging between the other uridines, thus filling the major groove. A molecular dynamics simulation indicates that these two conformations of the signature sequence represent stable alternatives. Analyses of the interaction with the HEXIM protein confirm the importance of the triple interactions at the signature sequence. Altogether, the present structural analysis of 7SK HP1 highlights an original mechanism of swapping bases, which could represent a possible '7SK signature' and provides new insight into the functional importance of the plasticity of RNA.",Date:"2016","Date Added":"2018-03-09 17:08:16","Date Modified":"2022-03-08 08:55:34","Access Date":null,Pages:"3568-3579","Num Pages":null,Issue:"6",Volume:45,"Number Of Volumes":null,"Journal Abbreviation":null,"Short Title":null,Series:null,"Series Number":null,"Series Text":null,"Series Title":null,Publisher:null,Place:null,Language:null,Rights:"All rights reserved",Type:null,Archive:null,"Archive Location":null,"Library Catalog":null,"Call Number":null,Extra:"PMID: 28082395",Notes:null,"File Attachments":"/Users/Tristan/Zotero/storage/5XS5T9UT/Martinez-Zapien et al._2016.pdf","Link Attachments":null,"Manual Tags":null,"Automatic Tags":null,Editor:null,"Series Editor":null,Translator:null,Contributor:null,"Attorney Agent":null,"Book Author":null,"Cast Member":null,Commenter:null,Composer:null,Cosponsor:null,Counsel:null,Interviewer:null,Producer:null,Recipient:null,"Reviewed Author":null,Scriptwriter:null,"Words By":null,Guest:null,Number:null,Edition:null,"Running Time":null,Scale:null,Medium:null,"Artwork Size":null,"Filing Date":null,"Application Number":null,Assignee:null,"Issuing Authority":null,Country:null,"Meeting Name":null,"Conference Name":null,Court:null,References:null,Reporter:null,"Legal Status":null,"Priority Numbers":null,"Programming Language":null,Version:null,System:null,Code:null,"Code Number":null,Section:null,Session:null,Committee:null,History:null,"Legislative Body":null,authors:"Martinez-Zapien, Denise; Legrand, Pierre; McEwen, Alastair G. A.G.; Proux, Florence; Cragnolini, Tristan; Pasquali, Samuela; Dock-Bregeon, Anne Catherine A.-C."},{Key:"XKRZHTMR","Item Type":"journalArticle","Publication Year":2015,Author:"Cragnolini, Tristan; Derreumaux, Philippe; Pasquali, Samuela",Title:"Ab initio RNA folding","Publication Title":"Journal of Physics: Condensed Matter",ISBN:null,ISSN:"0953-8984",DOI:"10.1088/0953-8984/27/23/233102",Url:"http://stacks.iop.org/0953-8984/27/i=23/a=233102?key=crossref.59918696e8eda8ad76d1c0736afdba62","Abstract Note":"© 2015 IOP Publishing Ltd. RNA molecules are essential cellular machines performing a wide variety of functions for which a specific three-dimensional structure is required. Over the last several years, the experimental determination of RNA structures through x-ray crystallography and NMR seems to have reached a plateau in the number of structures resolved each year, but as more and more RNA sequences are being discovered, the need for structure prediction tools to complement experimental data is strong. Theoretical approaches to RNA folding have been developed since the late nineties, when the first algorithms for secondary structure prediction appeared. Over the last 10 years a number of prediction methods for 3D structures have been developed, first based on bioinformatics and data-mining, and more recently based on a coarse-grained physical representation of the systems. In this review we are going to present the challenges of RNA structure prediction and the main ideas behind bioinformatic approaches and physics-based approaches. We will focus on the description of the more recent physics-based phenomenological models and on how they are built to include the specificity of the interactions of RNA bases, whose role is critical in folding. Through examples from different models, we will point out the strengths of physics-based approaches, which are able not only to predict equilibrium structures, but also to investigate dynamical and thermodynamical behavior, and the open challenges to include more key interactions ruling RNA folding.",Date:"2015-06-17","Date Added":"2018-03-02 12:16:36","Date Modified":"2018-04-24 11:25:16","Access Date":null,Pages:"233102","Num Pages":null,Issue:"23",Volume:27,"Number Of Volumes":null,"Journal Abbreviation":null,"Short Title":null,Series:null,"Series Number":null,"Series Text":null,"Series Title":null,Publisher:null,Place:null,Language:null,Rights:"All rights reserved",Type:null,Archive:null,"Archive Location":null,"Library Catalog":null,"Call Number":null,Extra:"Citation Key: Cragnolini2015",Notes:null,"File Attachments":"/Users/Tristan/Zotero/storage/6TPQC6IJ/Cragnolini, Derreumaux, Pasquali_2015.pdf","Link Attachments":null,"Manual Tags":null,"Automatic Tags":"RNA; force field; coarse-grained models",Editor:null,"Series Editor":null,Translator:null,Contributor:null,"Attorney Agent":null,"Book Author":null,"Cast Member":null,Commenter:null,Composer:null,Cosponsor:null,Counsel:null,Interviewer:null,Producer:null,Recipient:null,"Reviewed Author":null,Scriptwriter:null,"Words By":null,Guest:null,Number:null,Edition:null,"Running Time":null,Scale:null,Medium:null,"Artwork Size":null,"Filing Date":null,"Application Number":null,Assignee:null,"Issuing Authority":null,Country:null,"Meeting Name":null,"Conference Name":null,Court:null,References:null,Reporter:null,"Legal Status":null,"Priority Numbers":null,"Programming Language":null,Version:null,System:null,Code:null,"Code Number":null,Section:null,Session:null,Committee:null,History:null,"Legislative Body":null,authors:"Cragnolini, Tristan; Derreumaux, Philippe; Pasquali, Samuela"},{Key:"QERR75XD","Item Type":"journalArticle","Publication Year":2015,Author:"Cragnolini, Tristan; Laurin, Yoann; Derreumaux, Philippe; Pasquali, Samuela",Title:"Coarse-Grained HiRE-RNA Model for ab Initio RNA Folding beyond Simple Molecules, Including Noncanonical and Multiple Base Pairings","Publication Title":"Journal of Chemical Theory and Computation",ISBN:null,ISSN:"15499626",DOI:"10.1021/acs.jctc.5b00200",Url:"http://pubs.acs.org/doi/abs/10.1021/acs.jctc.5b00200","Abstract Note":"HiRE-RNA is a coarse-grained model for RNA structure prediction and the dynamical study of RNA folding. Using a reduced set of particles and detailed interactions accounting for base-pairing and stacking, we show that noncanonical and multiple base interactions are necessary to capture the full physical behavior of complex RNAs. In this paper, we give a full account of the model and present results on the folding, stability, and free energy surfaces of 16 systems with 12 to 76 nucleotides of increasingly complex architectures, ranging from monomers to dimers, using a total of 850 ?s of simulation time.\\nHiRE-RNA is a coarse-grained model for RNA structure prediction and the dynamical study of RNA folding. Using a reduced set of particles and detailed interactions accounting for base-pairing and stacking, we show that noncanonical and multiple base interactions are necessary to capture the full physical behavior of complex RNAs. In this paper, we give a full account of the model and present results on the folding, stability, and free energy surfaces of 16 systems with 12 to 76 nucleotides of increasingly complex architectures, ranging from monomers to dimers, using a total of 850 ?s of simulation time.",Date:"2015","Date Added":"2018-03-02 12:16:37","Date Modified":"2018-08-21 14:17:22","Access Date":null,Pages:"3510-3522","Num Pages":null,Issue:"7",Volume:11,"Number Of Volumes":null,"Journal Abbreviation":null,"Short Title":null,Series:null,"Series Number":null,"Series Text":null,"Series Title":null,Publisher:null,Place:null,Language:null,Rights:"All rights reserved",Type:null,Archive:null,"Archive Location":null,"Library Catalog":null,"Call Number":null,Extra:"Citation Key: Cragnolini2015a",Notes:null,"File Attachments":"/Users/Tristan/Zotero/storage/NH55X4HK/Cragnolini et al._2015.pdf; /Users/Tristan/Zotero/storage/MX3T8ED3/Cragnolini et al._2015(2).pdf","Link Attachments":null,"Manual Tags":null,"Automatic Tags":null,Editor:null,"Series Editor":null,Translator:null,Contributor:null,"Attorney Agent":null,"Book Author":null,"Cast Member":null,Commenter:null,Composer:null,Cosponsor:null,Counsel:null,Interviewer:null,Producer:null,Recipient:null,"Reviewed Author":null,Scriptwriter:null,"Words By":null,Guest:null,Number:null,Edition:null,"Running Time":null,Scale:null,Medium:null,"Artwork Size":null,"Filing Date":null,"Application Number":null,Assignee:null,"Issuing Authority":null,Country:null,"Meeting Name":null,"Conference Name":null,Court:null,References:null,Reporter:null,"Legal Status":null,"Priority Numbers":null,"Programming Language":null,Version:null,System:null,Code:null,"Code Number":null,Section:null,Session:null,Committee:null,History:null,"Legislative Body":null,authors:"Cragnolini, Tristan; Laurin, Yoann; Derreumaux, Philippe; Pasquali, Samuela"},{Key:"DD4DEV26","Item Type":"journalArticle","Publication Year":2015,Author:"Laurent, Benoist; Chavent, Matthieu; Cragnolini, Tristan; Dahl, Anna Caroline E.; Pasquali, Samuela; Derreumaux, Philippe; Sansom, Mark S.P.; Baaden, Marc",Title:"Epock: Rapid analysis of protein pocket dynamics","Publication Title":"Bioinformatics",ISBN:null,ISSN:"14602059",DOI:"10.1093/bioinformatics/btu822",Url:"https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btu822","Abstract Note":"SUMMARY: The volume of an internal protein pocket is fundamental to ligand accessibility. Few programs that compute such volumes manage dynamic data from Molecular Dynamics (MD) simulations. Limited performance often prohibits analysis of large datasets. We present Epock, an efficient command-line tool that calculates pocket volumes from MD trajectories. A plugin for the VMD program provides a graphical user interface to facilitate input creation, run Epock and analyse the results. Availability and Implementation: Epock C++ source code, Python analysis scripts, VMD Tcl plugin, documentation and installation instructions are freely available at http://epock.bitbucket.org. CONTACT: benoist.laurent@gmail.com, baaden@smplinux.de.",Date:"2015-05-01","Date Added":"2018-03-09 17:08:16","Date Modified":"2018-04-05 14:55:35","Access Date":null,Pages:"1478-1480","Num Pages":null,Issue:"9",Volume:31,"Number Of Volumes":null,"Journal Abbreviation":null,"Short Title":null,Series:null,"Series Number":null,"Series Text":null,"Series Title":null,Publisher:null,Place:null,Language:null,Rights:"All rights reserved",Type:null,Archive:null,"Archive Location":null,"Library Catalog":null,"Call Number":null,Extra:"PMID: 25505095 ISBN: 1367-4811 (Electronic)\\r1367-4803 (Linking)",Notes:null,"File Attachments":"/Users/Tristan/Zotero/storage/2PGSCXQ3/Laurent et al._2015.pdf; /Users/Tristan/Zotero/storage/ZDFUCE6C/Laurent et al._2015(2).pdf","Link Attachments":null,"Manual Tags":null,"Automatic Tags":null,Editor:null,"Series Editor":null,Translator:null,Contributor:null,"Attorney Agent":null,"Book Author":null,"Cast Member":null,Commenter:null,Composer:null,Cosponsor:null,Counsel:null,Interviewer:null,Producer:null,Recipient:null,"Reviewed Author":null,Scriptwriter:null,"Words By":null,Guest:null,Number:null,Edition:null,"Running Time":null,Scale:null,Medium:null,"Artwork Size":null,"Filing Date":null,"Application Number":null,Assignee:null,"Issuing Authority":null,Country:null,"Meeting Name":null,"Conference Name":null,Court:null,References:null,Reporter:null,"Legal Status":null,"Priority Numbers":null,"Programming Language":null,Version:null,System:null,Code:null,"Code Number":null,Section:null,Session:null,Committee:null,History:null,"Legislative Body":null,authors:"Laurent, Benoist; Chavent, Matthieu; Cragnolini, Tristan; Dahl, Anna Caroline E.; Pasquali, Samuela; Derreumaux, Philippe; Sansom, Mark S.P.; Baaden, Marc"},{Key:"ICYH2PYH","Item Type":"conferencePaper","Publication Year":2015,Author:"Doutreligne, Sebastien; Gageat, Cedric; Cragnolini, Tristan; Taly, Antoine; Pasquali, Samuela; Derreumaux, Philippe; Baaden, Marc",Title:"UnityMol: interactive and ludic visual manipulation of coarse-grained RNA and other biomolecules","Publication Title":"2015 IEEE 1st International Workshop on Virtual and Augmented Reality for Molecular Science (VARMS@IEEEVR)",ISBN:"978-1-4673-6926-8",ISSN:null,DOI:"10.1109/VARMS.2015.7151718",Url:"http://ieeexplore.ieee.org/document/7151718/","Abstract Note":"© 2015 IEEE.We present a general software architecture to carry out interactive molecular simulations in a game engine environment. Our implementation is based on the UnityMol framework and the HireRNA physics engine. With UnityMol, we pursue the goal to create an interactive virtual laboratory enabling researchers in biology to visualize biomolecular systems, run simulations and interact with physical models and data. Similarly, UnityMol enables game designers to build scientifically accurate molecular scenarios. We discuss four case studies, from simulation setup via immersive experiments, force-induced unfolding of RNA to teaching and collaborative research applications. Visual effects enrich the dynamic and immersive aspects. We combine an appealing visual feedback with a set of analysis features to extract information about properties of the fascinating biomolecular systems under study. Access to various input devices enables a natural interaction with the simulation.",Date:"2015-03-24","Date Added":"2017-01-05 17:05:45","Date Modified":"2022-03-14 11:57:17","Access Date":null,Pages:"1-6","Num Pages":null,Issue:null,Volume:null,"Number Of Volumes":null,"Journal Abbreviation":null,"Short Title":null,Series:null,"Series Number":null,"Series Text":null,"Series Title":null,Publisher:"IEEE",Place:null,Language:null,Rights:"All rights reserved",Type:null,Archive:null,"Archive Location":null,"Library Catalog":null,"Call Number":null,Extra:null,Notes:null,"File Attachments":null,"Link Attachments":null,"Manual Tags":null,"Automatic Tags":"[I.6.6 [simulation and modeling]-simulation output",Editor:null,"Series Editor":null,Translator:null,Contributor:null,"Attorney Agent":null,"Book Author":null,"Cast Member":null,Commenter:null,Composer:null,Cosponsor:null,Counsel:null,Interviewer:null,Producer:null,Recipient:null,"Reviewed Author":null,Scriptwriter:null,"Words By":null,Guest:null,Number:null,Edition:null,"Running Time":null,Scale:null,Medium:null,"Artwork Size":null,"Filing Date":null,"Application Number":null,Assignee:null,"Issuing Authority":null,Country:null,"Meeting Name":null,"Conference Name":null,Court:null,References:null,Reporter:null,"Legal Status":null,"Priority Numbers":null,"Programming Language":null,Version:null,System:null,Code:null,"Code Number":null,Section:null,Session:null,Committee:null,History:null,"Legislative Body":null,authors:"Doutreligne, Sebastien; Gageat, Cedric; Cragnolini, Tristan; Taly, Antoine; Pasquali, Samuela; Derreumaux, Philippe; Baaden, Marc"},{Key:"PE5PMAIG","Item Type":"conferencePaper","Publication Year":2014,Author:"Doutreligne, Sebastien; Cragnolini, Tristan; Pasquali, Samuela; Derreumaux, Philippe; Baaden, Marc",Title:"UnityMol: Interactive scientific visualization for integrative biology","Publication Title":"2014 IEEE 4th Symposium on Large Data Analysis and Visualization (LDAV)",ISBN:"978-1-4799-5215-1",ISSN:null,DOI:"10.1109/LDAV.2014.7013213",Url:"http://ieeexplore.ieee.org/document/7013213/","Abstract Note":"© 2014 IEEE.A broad challenge facing scientists today is the availability of huge amounts of data from various sources. Computers are required to store, analyze, explore and represent these data in order to extract useful information. With UnityMol, we pursue the ambitious goal to create an interactive virtual laboratory enabling researchers in biology to visualize biomolecular systems, run simulations and interact with physical models and data. Visual effects can enrich the dynamic and immersive aspects. Ultimately, we want to combine an appealing visual feedback already in place with a set of analysis features to extract information about properties of the fascinating biomolecular systems under study.",Date:"2014-11","Date Added":"2017-01-05 17:05:45","Date Modified":"2022-03-14 11:57:17","Access Date":null,Pages:"109-110","Num Pages":null,Issue:null,Volume:null,"Number Of Volumes":null,"Journal Abbreviation":null,"Short Title":null,Series:null,"Series Number":null,"Series Text":null,"Series Title":null,Publisher:"IEEE",Place:null,Language:null,Rights:"All rights reserved",Type:null,Archive:null,"Archive Location":null,"Library Catalog":null,"Call Number":null,Extra:null,Notes:null,"File Attachments":"/Users/Tristan/Zotero/storage/RCR9CPTM/Doutreligne et al._2014.pdf","Link Attachments":null,"Manual Tags":null,"Automatic Tags":"[Biology and Genetics; I.6.6 [Simulation and Model",Editor:null,"Series Editor":null,Translator:null,Contributor:null,"Attorney Agent":null,"Book Author":null,"Cast Member":null,Commenter:null,Composer:null,Cosponsor:null,Counsel:null,Interviewer:null,Producer:null,Recipient:null,"Reviewed Author":null,Scriptwriter:null,"Words By":null,Guest:null,Number:null,Edition:null,"Running Time":null,Scale:null,Medium:null,"Artwork Size":null,"Filing Date":null,"Application Number":null,Assignee:null,"Issuing Authority":null,Country:null,"Meeting Name":null,"Conference Name":null,Court:null,References:null,Reporter:null,"Legal Status":null,"Priority Numbers":null,"Programming Language":null,Version:null,System:null,Code:null,"Code Number":null,Section:null,Session:null,Committee:null,History:null,"Legislative Body":null,authors:"Doutreligne, Sebastien; Cragnolini, Tristan; Pasquali, Samuela; Derreumaux, Philippe; Baaden, Marc"},{Key:"46VRXFSY","Item Type":"journalArticle","Publication Year":2014,Author:"Sterpone, Fabio; Melchionna, Simone; Tuffery, Pierre; Pasquali, Samuela; Mousseau, Normand; Cragnolini, Tristan; Chebaro, Yassmine; St-Pierre, Jean-Francois; Kalimeri, Maria; Barducci, Alessandro; Laurin, Yoann; Tek, Alex; Baaden, Marc; Nguyen, Phuong Hoang; Derreumaux, Philippe",Title:"The OPEP protein model: from single molecules, amyloid formation, crowding and hydrodynamics to DNA/RNA systems","Publication Title":"Chem. Soc. Rev.",ISBN:null,ISSN:"0306-0012",DOI:"10.1039/C4CS00048J",Url:"http://www.ncbi.nlm.nih.gov/pubmed/24759934","Abstract Note":"The OPEP coarse-grained protein model has been applied to a wide range of applications since its first release 15 years ago.",Date:"2014-07-07","Date Added":"2018-03-02 12:16:36","Date Modified":"2022-03-14 11:57:17","Access Date":"2014-06-29",Pages:"4871-4893","Num Pages":null,Issue:"13",Volume:43,"Number Of Volumes":null,"Journal Abbreviation":null,"Short Title":null,Series:null,"Series Number":null,"Series Text":null,"Series Title":null,Publisher:null,Place:null,Language:null,Rights:"All rights reserved",Type:null,Archive:null,"Archive Location":null,"Library Catalog":null,"Call Number":null,Extra:"PMID: 24759934 Citation Key: Sterpone2014",Notes:null,"File Attachments":"; /Users/Tristan/Zotero/storage/G8DXXYC6/Sterpone et al._2014.pdf","Link Attachments":"http://xlink.rsc.org/?DOI=C4CS00048J","Manual Tags":null,"Automatic Tags":null,Editor:null,"Series Editor":null,Translator:null,Contributor:null,"Attorney Agent":null,"Book Author":null,"Cast Member":null,Commenter:null,Composer:null,Cosponsor:null,Counsel:null,Interviewer:null,Producer:null,Recipient:null,"Reviewed Author":null,Scriptwriter:null,"Words By":null,Guest:null,Number:null,Edition:null,"Running Time":null,Scale:null,Medium:null,"Artwork Size":null,"Filing Date":null,"Application Number":null,Assignee:null,"Issuing Authority":null,Country:null,"Meeting Name":null,"Conference Name":null,Court:null,References:null,Reporter:null,"Legal Status":null,"Priority Numbers":null,"Programming Language":null,Version:null,System:null,Code:null,"Code Number":null,Section:null,Session:null,Committee:null,History:null,"Legislative Body":null,authors:"Sterpone, Fabio; Melchionna, Simone; Tuffery, Pierre; Pasquali, Samuela; Mousseau, Normand; Cragnolini, Tristan; Chebaro, Yassmine; St-Pierre, Jean-Francois; Kalimeri, Maria; Barducci, Alessandro; Laurin, Yoann; Tek, Alex; Baaden, Marc; Nguyen, Phuong Hoang; Derreumaux, Philippe"},{Key:"GZC5HUA6","Item Type":"journalArticle","Publication Year":2014,Author:"Tiwari, Sandhya P; Fuglebakk, Edvin; Hollup, Siv M; Skjærven, Lars; Cragnolini, Tristan; Grindhaug, Svenn H; Tekle, Kidane M; Reuter, Nathalie",Title:"WEBnm@ v2.0: Web server and services for comparing protein flexibility","Publication Title":"BMC Bioinformatics",ISBN:null,ISSN:"1471-2105",DOI:"10.1186/s12859-014-0427-6",Url:"http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-014-0427-6","Abstract Note":"© 2015 Tiwari et al.; licensee BioMed Central.Background: Normal mode analysis (NMA) using elastic network models is a reliable and cost-effective computational method to characterise protein flexibility and by extension, their dynamics. Further insight into the dynamics-function relationship can be gained by comparing protein motions between protein homologs and functional classifications. This can be achieved by comparing normal modes obtained from sets of evolutionary related proteins. Results: We have developed an automated tool for comparative NMA of a set of pre-aligned protein structures. The user can submit a sequence alignment in the FASTA format and the corresponding coordinate files in the Protein Data Bank (PDB) format. The computed normalised squared atomic fluctuations and atomic deformation energies of the submitted structures can be easily compared on graphs provided by the web user interface. The web server provides pairwise comparison of the dynamics of all proteins included in the submitted set using two measures: the Root Mean Squared Inner Product and the Bhattacharyya Coefficient. The Comparative Analysis has been implemented on our web server for NMA, WEBnmat, which also provides recently upgraded functionality for NMA of single protein structures. This includes new visualisations of protein motion, visualisation of inter-residue correlations and the analysis of conformational change using the. In addition, programmatic access to WEBnmat is now available through a SOAP-based web service. WEBnmat is available at. Conclusion: WEBnmat v2.0 is an online tool offering unique capability for comparative NMA on multiple protein structures. Along with a convenient web interface, powerful computing resources, and several methods for mode analyses, WEBnmat facilitates the assessment of protein flexibility within protein families and superfamilies. These analyses can give a good view of how the structures move and how the flexibility is conserved over the different structures.",Date:"2014-12-30","Date Added":"2019-09-05 14:03:49","Date Modified":"2022-03-14 11:57:17","Access Date":null,Pages:"427","Num Pages":null,Issue:"1",Volume:15,"Number Of Volumes":null,"Journal Abbreviation":null,"Short Title":null,Series:null,"Series Number":null,"Series Text":null,"Series Title":null,Publisher:null,Place:null,Language:null,Rights:"All rights reserved",Type:null,Archive:null,"Archive Location":null,"Library Catalog":null,"Call Number":null,Extra:null,Notes:null,"File Attachments":"; /Users/Tristan/Zotero/storage/7RT25A39/Tiwari et al. - 2015 - WEBnm @ v2 . 0 Web server and services for comparing protein flexibility.pdf","Link Attachments":"https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-014-0427-6","Manual Tags":null,"Automatic Tags":"Normal mode analysis; [Adenylate kinase; adenylate kinase; bhattacharyya coefficient; Bhattacharyya coefficient; Comp; Compar; comparative analysis; elastic network models; Elastic network models; normal mode analysis; tim barrels; web-tool",Editor:null,"Series Editor":null,Translator:null,Contributor:null,"Attorney Agent":null,"Book Author":null,"Cast Member":null,Commenter:null,Composer:null,Cosponsor:null,Counsel:null,Interviewer:null,Producer:null,Recipient:null,"Reviewed Author":null,Scriptwriter:null,"Words By":null,Guest:null,Number:null,Edition:null,"Running Time":null,Scale:null,Medium:null,"Artwork Size":null,"Filing Date":null,"Application Number":null,Assignee:null,"Issuing Authority":null,Country:null,"Meeting Name":null,"Conference Name":null,Court:null,References:null,Reporter:null,"Legal Status":null,"Priority Numbers":null,"Programming Language":null,Version:null,System:null,Code:null,"Code Number":null,Section:null,Session:null,Committee:null,History:null,"Legislative Body":null,authors:"Tiwari, Sandhya P; Fuglebakk, Edvin; Hollup, Siv M; Skjærven, Lars; Cragnolini, Tristan; Grindhaug, Svenn H; Tekle, Kidane M; Reuter, Nathalie"},{Key:"A7YDZXDW","Item Type":"journalArticle","Publication Year":2013,Author:"Cragnolini, Tristan; Derreumaux, Philippe; Pasquali, Samuela",Title:"Coarse-Grained Simulations of RNA and DNA Duplexes","Publication Title":"The Journal of Physical Chemistry B",ISBN:null,ISSN:"1520-6106",DOI:"10.1021/jp400786b",Url:"http://www.ncbi.nlm.nih.gov/pubmed/23730911","Abstract Note":"Although RNAs play many cellular functions, little is known about the dynamics and thermodynamics of these molecules. In principle, all-atom molecular dynamics simulations can investigate these issues, but with current computer facilities, these simulations have been limited to small RNAs and to short times. HiRe-RNA, a recently proposed high-resolution coarse-grained RNA that captures many geometric details such as base pairing and stacking, is able to fold RNA molecules to near-native structures in a short computational time. So far, it had been applied to simple hairpins, and here we present its application to duplexes of a couple dozen nucleotides and show how with replica exchange molecular dynamics (REMD) we can easily predict the correct double helix from a completely random configuration and study the dissociation curve. To show the versatility of our model, we present an application to a double stranded DNA molecule as well. A reconstruction algorithm allows us to obtain full atom structures from the coarse-grained model. Through atomistic molecular dynamics (MD), we can compare the dynamics starting from a representative structure of a low temperature replica or from the experimental structure, and show how the two are statistically identical, highlighting the validity of a coarse-grained approach for structured RNAs and DNAs.",Date:"2013-07-11","Date Added":"2018-03-02 12:16:36","Date Modified":"2022-07-03 17:41:33","Access Date":null,Pages:"8047-8060","Num Pages":null,Issue:"27",Volume:117,"Number Of Volumes":null,"Journal Abbreviation":null,"Short Title":null,Series:null,"Series Number":null,"Series Text":null,"Series Title":null,Publisher:null,Place:null,Language:null,Rights:"All rights reserved",Type:null,Archive:null,"Archive Location":null,"Library Catalog":null,"Call Number":null,Extra:"PMID: 23730911 Citation Key: Cragnolini2013",Notes:null,"File Attachments":"; /Users/Tristan/Zotero/storage/TK3T69LR/Cragnolini, Derreumaux, Pasquali_2013.pdf","Link Attachments":"https://pubs.acs.org/doi/10.1021/jp400786b","Manual Tags":null,"Automatic Tags":null,Editor:null,"Series Editor":null,Translator:null,Contributor:null,"Attorney Agent":null,"Book Author":null,"Cast Member":null,Commenter:null,Composer:null,Cosponsor:null,Counsel:null,Interviewer:null,Producer:null,Recipient:null,"Reviewed Author":null,Scriptwriter:null,"Words By":null,Guest:null,Number:null,Edition:null,"Running Time":null,Scale:null,Medium:null,"Artwork Size":null,"Filing Date":null,"Application Number":null,Assignee:null,"Issuing Authority":null,Country:null,"Meeting Name":null,"Conference Name":null,Court:null,References:null,Reporter:null,"Legal Status":null,"Priority Numbers":null,"Programming Language":null,Version:null,System:null,Code:null,"Code Number":null,Section:null,Session:null,Committee:null,History:null,"Legislative Body":null,authors:"Cragnolini, Tristan; Derreumaux, Philippe; Pasquali, Samuela"}];export{e as p};
